test/functional/sqlsrv/TC44_FetchArray.phpt

```diff
@@ -6,7 +6,16 @@ by checking all fetch type modes.
 --ENV--
 PHPT_EXEC=true
 --SKIPIF--
-<?php require('skipif_versions_old.inc'); ?>
+<?
+// locale must be set before 1st connection
+if ( !isWindows() ) {
+    setlocale(LC_ALL, "en_US.ISO-8859-1");
+}
+
+// this skips for older ODBC versions in Linux which doesn't support non-UTF8
+setTestAnsiData(true)
+php require('skipif_versions_old.inc');
```

Comment:
why do you call setTestAnsiData() in SKIPIF section?

Reply:
it's needed by the skipif